instructions/bundling-scripts/zimbra-store.sh

```diff
@@ -48,7 +48,6 @@ main()
        cp -f ${repoDir}/zm-sync-tools/src/bin/zmsyncreverseproxy ${repoDir}/zm-build/${currentPackage}/opt/zimbra/bin/zmsyncreverseproxy
        cp -f ${repoDir}/zm-sync-store/src/bin/zmdevicesstats ${repoDir}/zm-build/${currentPackage}/opt/zimbra/bin/zmdevicesstats
        cp -f ${repoDir}/zm-sync-store/src/bin/zmgdcutil ${repoDir}/zm-build/${currentPackage}/opt/zimbra/bin/zmgdcutil
-       cp -f ${repoDir}/zm-sync-store/src/bin/zmmdmnotificationmail ${repoDir}/zm-build/${currentPackage}/opt/zimbra/bin/zmmdmnotificationmail
```

Comment:
have we removed this binary from zm-sync-store repo?

Reply:
https://github.com/Zimbra/zm-sync-store/pull/84